frontend-react/package.json

```diff
@@ -228,5 +228,8 @@
     "resolutions": {
         "@types/react": "<18.0.0",
         "@types/react-dom": "<18.0.0"
+    },
+    "engines": {
+      "node": "~14.20.0"
```

Comment:
Possible conflict with .1 used elsewhere?

Reply:
@doug-s-nava this allows "approximate" version, which will get 14.20.1 for now, but may get newer versions in the future.  Will that conflict with the hard coded 14.20.1 used elsewhere?

Reply:
in my mind it is fine / preferable to be more permissive here. The real source of truth for node version in use will be the nvmrc, any github action specs, and hopefully down the line, dockerfiles. This engines definition will cause npm installs to error if a version outside of the one specified is used, but if all of the above mentioned files are pinned more specifically, as they should be, that will be fine. This just gives local developers a little bit more leeway, not forcing them to install new patch versions all the time to keep up.

On a procedural note, I'd love to have this wrapped up this morning, as Monday is my last day and I'd like to get the team onboarded to nvm before I leave